client/my-sites/patterns/components/header/style.scss

```
@@ -8,7 +8,7 @@
 
 	.patterns-header__inner {
 		padding: 105px 0 64px 84px;
-		background: url(./images/background.svg) no-repeat right bottom;
+		background: url(./images/background@3x.png) no-repeat bottom right 84px / auto 100%;
```

```suggestion
		background-image: url(./images/background@3x.png);
		background-repeat: no-repeat;
		background-position: bottom right 84px;
		background-size: auto 100%;
```
Not a blocker, but I advocate using the non-shorthand rules in the source code for improved readability. Especially in cases like this where we specify so many rules.

---

🙌 I agree!

I continued with the shorthand here as I thought that was a convention we used. I much prefer the individual rules though.